assets/app/views/settings.html

```diff
@@ -2,8 +2,17 @@
   <project-page>
     <div ng-controller="SettingsController">
       <div class="page-header page-header-bleed-right">
+        <div class="actions action-button pull-right">
+          <i class="fa fa-trash-o fa-2x" 
+            ng-click="openDeleteModal()"
+            ng-if="!deletingMap[project.metadata.name] && !confirmingMap[project.metadata.name]" 
```

Comment:
i think this was copied by mistake, deletingMap and confirmingMap dont exist on the settings page